### DISCUSSION TITLE
Scammers

### DISCUSSION THREAD

**Discussion by EagleBestDeals** [community/community#126273] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello guys,

Be careful with **NopeCha**. I have been charged for 9 months in a row straight and i never used the credits, nopeCha was not working, but my subscription on paypal kept charging me. I messaged nopeCha's support both on discord and email, i didn't get any answers for a month. After their latest update that they announced on discord, i messaged them again, asking what is going on and why am i not getting answers, they blocked me from discord and i got no answers at all, nor on there nor to my email. They could've said that i will not get refunded, or that they will let me have the next few months for free due to that and so on. They simply blocked me without saying a single word. Be careful with such scam services that are charging you due to unwanted subscriptions. I had the same issue with another company AlfredCamera and i received a refund for every single month that i paid but not used it. NopeCha can see that i had 2k/day calls that i never used, but woah, i guess blocking is better than giving any answer.
![image](https://github.com/community/community/assets/27913600/481d508a-4066-4977-9c7c-851e982110e7)
Here are my emails with their support. See the charges and my messages, no answers.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Awkword98, Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
